### ISSUE TITLE
MultiInput NVDA screen reader issue

### ISSUE THREAD

**Issue by mdasari84** [SAP/ui5-webcomponents#5322]
### **Bug Description**
NVDA screen reader doesn't read the tokens on the focus of the MultiInput field. It reads "Multi Value Input read only Contains X tokens blank" 
Note: X is number of the tokens

### **Expected Behavior**
NVDA screen reader should read the tokens on the focus of the MultiInput field

### **Steps to Reproduce**
1. Go to https://codesandbox.io/s/ui5-wc-multiinput-accessibility-issue-w34p63?file=/index.html
2. Open NVDA, use the keyboard to focus on the MultiInput field
3. NVDA reads "Multi Value Input read only Contains 3 tokens blank" instead of reading the token values

### **Isolated Example**

https://codesandbox.io/s/ui5-wc-multiinput-accessibility-issue-w34p63?file=/index.html

### **Context**
- UI5 Web Components version: 1.3.x
- OS/Platform: Windows
- Browser: Chrome
- Affected component: MultiInput

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors
- Business impact: {...}

**Comment by niyap** [comment]
Hello @mdasari84,

Thanks for sharing your findings!

That is the expected speech output when the focus is in the input field. When you navigate to a token via arrow keys, its text is read out as expected.
Here, you can check: https://sap.github.io/ui5-webcomponents/main/playground/main/pages/MultiInput/?sap-ui-theme=sap_horizon

Kind Regards,
Niya
